roles/ansible-test/tasks/init_test_options.yaml

```diff
@@ -4,6 +4,11 @@
     ansible_test_options: "{{ ansible_test_options }} --skip-tags {{ ansible_test_skip_tags }}"
   when: ansible_test_skip_tags is defined
 
+- name: Setup --tags for test_options
```

Comment:
Lets do negative match here, for other platforms the tags were not all setup propelry.